monitors/ContractMonitor.js

```diff
@@ -140,16 +140,18 @@ class ContractMonitor {
         createEtherscanLinkMarkdown(this.web3, event.liquidator) +
         (this.monitoredLiquidators.indexOf(event.liquidator) != -1 ? " (Monitored liquidator bot)" : "") +
         " initiated liquidation for " +
+        this.formatDecimalString(event.lockedCollateral) +
+        " (liquidated collateral = " +
         this.formatDecimalString(event.liquidatedCollateral) +
-        " " +
+        ") " +
         this.collateralCurrencySymbol +
         " of sponsor " +
         createEtherscanLinkMarkdown(this.web3, event.sponsor) +
         " collateral backing " +
         this.formatDecimalString(event.tokensOutstanding) +
         " " +
         this.syntheticCurrencySymbol +
-        " tokens. Sponsor collateralization was " +
+        " tokens. Sponsor collateralization (based on 'liquidated' not 'locked' collateral) was " +
```

Comment:
This is great. the only addition we could consider here (not for this PR) would be to include the price that the liquidation occurred at.

Reply:
Agreed. I left this out because the price is displayed on the Liquidation bot's logs, but might as well add it here too